src/components/SignUp.js

```diff
@@ -63,12 +65,33 @@ const useStyles = makeStyles((theme) => ({
 
 export default function SignUp() {
   const classes = useStyles();
-  const { handleSubmit, control, formState: { errors } } = useForm();
+  const { handleSubmit, control, formState: { errors }, getValues } = useForm();
 
-  const onSubmit = (data) => {
-    // Additional actions on form submission here
-    // placemarker to call our API to chk and save data
-    console.log(data);
+  const [errorMessage, setErrorMessage] = useState(null);
+  
+  const onSubmit = async (data) => {
+    // console.log(data);
```

Comment:
Lets clean this up.

Reply:
I'll go through and check for any unneeded comments/commented out stuff when I do the redirect (to SignIn?).